### PR TITLE
implemented login endpoint and fixed bug in registration (pw not hashed)

### PR DESCRIPTION
**Pull request by tehvicke**
login endpoint fixed

### CODE REVIEW

File: Routes/user.js [hunk 1]
```diff
@@ -15,4 +16,26 @@ router.post('/registration', async (req, res, next) => {
   }
 })
 
+router.post('/login', async (req, res, next) => {
+  const { username, password } = req.body
+  console.log('1')
```
Comment (erikpantzar) [inline Routes/user.js:21]: Remove console log.

File: Routes/user.js [hunk 2]
```diff
@@ -15,4 +16,26 @@ router.post('/registration', async (req, res, next) => {
...
+  try {
+    const user = await User.findOne({ username: username })
+    console.log('2')
+    console.log(user)
```
Comment (erikpantzar) [inline Routes/user.js:25]: Remove console logs.

File: Routes/user.js [hunk 3]
```diff
@@ -15,4 +16,26 @@ router.post('/registration', async (req, res, next) => {
...
+    if (user && bcrypt.compareSync(password, user.password)) {
+      res.status(200).json(user.accessToken)
+    } else {
+      console.log('3')
```
Comment (erikpantzar) [inline Routes/user.js:29]: Remove console log.

**Comment by tehvicke** [comment]
removed console logs and fixed error

**Pull request merged**